.evergreen/run-benchmarks.sh

```diff
@@ -10,4 +10,11 @@ export MONGODB_URI=$MONGODB_URI
 export MONGODB_CLIENT_OPTIONS=$MONGODB_CLIENT_OPTIONS
 
 npm run build:ts
-npm run check:bench
+
+
+# If MONGODB_CLIENT_OPTIONS contains __enableMongoLogger redirect stderr to a file
+if [[ $MONGODB_CLIENT_OPTIONS == *"__enableMongoLogger"* ]]; then
+  npm run check:bench 2> bench.log
```

Comment:
The benchmarks generate about 3.5GB of logs :) we do not need to put that pressure on parsely. Plus, I imagine this is exactly how one might quickly get logs into a file they can upload somewhere, so this is more realistic than printing right to stderr.